BUILD.bazel

```diff
@@ -4,4 +4,8 @@ load("@bazel_gazelle//:def.bzl", "gazelle")
 # gazelle:resolve proto protoc-gen-swagger/options/annotations.proto @com_github_grpc_ecosystem_grpc_gateway//protoc-gen-swagger/options:options_proto
```

Comment:
A couple questions; 

- What's the ETA to check this in? 
- Could you provide a quick step by step overview (5/6 sentences) of how it works? What does the DSL provide? Where to the types come from? 
- Is it a temporary implementation to speed things up for the dev summit? Or will we do some changes later? (Let's add TODOs if the latter)